easybuild/easyconfigs/l/libdap/libdap-3.20.3-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'libdap'
+version = '3.20.3'
+
+homepage = 'http://opendap.org/download/libdap'
+description = """A C++ SDK which contains an implementation of DAP 2.0
```

Comment:
If it is a C++ SDK, then I think it should be one step higher up, i.e., in GCC/iccifort level.
C++ is notorious for being incompatible between compilers. Think outside of GCC/Intel, like PGI

Reply:
@smoors Probably just picked based on the existing `libdap-3.19.1-GCCcore-6.4.0.eb` (which we haven't had problems with yet, I think). So I'd say it's fine?

Maybe @vanzod has a suggestion?

Reply:
As far as I know we haven't seen any problems with this yet, so I'm calling this.

We can adjust accordingly if problems arise.